Cloudinary/Api.cs

```diff
@@ -77,7 +77,7 @@ static Api()
         /// <param name="parameters">Dictionary of call parameters (can be null)</param>
         /// <param name="file">File to upload (must be null for non-uploading actions)</param>
         /// <returns>HTTP response on call</returns>
-        public HttpWebResponse Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
+        public override object Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
```

Comment:
No, just make type casting when you call this method. We did this in all places where this method is called.
We'll check the functionality of the second one after the code is changed by test run.